core/templates/dev/head/expressions/expressionSyntaxTree.js

```diff
@@ -0,0 +1,426 @@
+             _verifyArgTypesMatchExpectedType(args, 'Real');
+             return 'UnicodeString';
+           },
+           eval: function(args) {
+             verifyNumArgs(args, 2);
+             var numericArgs = _coerceAllArgsToNumber(args);
+             return numericArgs[0] >= numericArgs[1];
+           }
+         },
+         '<': {
+           getType: function(args) {
+             verifyNumArgs(args, 2);
+             _verifyArgTypesMatchExpectedType(args, 'Real');
+             return 'UnicodeString';
+           },
+           eval: function(args) {
+             verifyNumArgs(args, 2);
+             var numericArgs = _coerceAllArgsToNumber(args);
+             return numericArgs[0] < numericArgs[1];
+           }
+         },
+         '>': {
+           getType: function(args) {
+             verifyNumArgs(args, 2);
+             _verifyArgTypesMatchExpectedType(args, 'Real');
+             return 'UnicodeString';
+           },
+           eval: function(args) {
+             verifyNumArgs(args, 2);
+             var numericArgs = _coerceAllArgsToNumber(args);
+             return numericArgs[0] > numericArgs[1];
+           }
+         },
+         '!': {
+           getType: function(args) {
+             verifyNumArgs(args, 1);
+             _verifyArgTypesMatchExpectedType(args, 'UnicodeString');
+             return 'UnicodeString';
+           },
+           eval: function(args) {
+             verifyNumArgs(args, 1);
+             return !args[0];
+           }
+         },
+         '==': {
+           getType: function(args) {
+             verifyNumArgs(args, 2);
+             return 'UnicodeString';
+           },
+           eval: function(args) {
+             verifyNumArgs(args, 2);
+             return args[0] == args[1];
+           }
+         },
+         '!=': {
+           getType: function(args) {
+             verifyNumArgs(args, 2);
+             return 'UnicodeString';
+           },
+           eval: function(args) {
+             verifyNumArgs(args, 2);
+             return args[0] != args[1];
+           }
+         },
+         '&&': {
+           getType: function(args) {
+             verifyNumArgs(args, 2);
+             _verifyArgTypesMatchExpectedType(args, 'UnicodeString');
+             return 'UnicodeString';
+           },
+           eval: function(args) {
+             // TODO(kashida): Make this short-circuit.
+             verifyNumArgs(args, 2);
+             return Boolean(args[0] && args[1]);
+           }
+         },
+         '||': {
+           getType: function(args) {
+             verifyNumArgs(args, 2);
+             _verifyArgTypesMatchExpectedType(args, 'UnicodeString');
+             return 'UnicodeString';
+           },
+           eval: function(args) {
+             // TODO(kashida): Make this short-circuit.
+             verifyNumArgs(args, 2);
+             return Boolean(args[0] || args[1]);
+           }
+         },
+         if: {
```

Comment:
Please replicate this comment:

    https://github.com/oppia/oppia/pull/1390/files#diff-31c7a4062b9906056a1f022cddfb23deL357

and introduce quotation marks for the 'if' and the next few strings.